apps/developer-hub/README.md

```diff
@@ -0,0 +1,136 @@
+# Pyth Developer Hub
+
+Welcome to the Pyth Developer Hub repository! This repository contains the documentation for Pyth Network's developer resources, including guides, tutorials, and API references.
+
+## Getting Started
+
+### Prerequisites
+
+- Node.js >= 22.14.0
+- pnpm
+
+### Setup
+
+1. Clone the monorepo repository
+2. Navigate to the developer hub directory:
+
+   ```bash
+   cd apps/developer-hub
+   ```
+
+3. Install dependencies:
+
+   ```bash
+   pnpm install
+   ```
+
+4. Build the project:
+
+   ```bash
+   pnpm run build
+   ```
+
+5. Start the development server:
+
+   ```bash
+   pnpm run start:dev
```

Comment:
ditto for this command (should be `pnpm run turbo start:dev` if run from this folder)